### ISSUE TITLE
bulk all download

### ISSUE THREAD

**Issue by flckv** [bilalazh/Arxiv-Research-Pooler#1]
great work. Did you try downloading all arxiv in a specific category e.g.: biology to local folder?

**Comment by bilalazh** [comment]
hello @flckv Thanks for showing interest in the project 

-- 
**goal of the project for me :** 
i made this project due to overwhelming arxiv papers that are added everyday in domains like cs.AI  , due to overwhelming amount i thought it would be nice to have some way  of seeing daily papers with abstracts that interest me and then downloading them in bulk (my bulk i mean 20 to 30 ) what i can read personally in a day 

---
**issues :** 
arxiv api was super unrealiable  , and that made me not work on this projec for the time being 
Here are two issues worth looking 

This issue i created 
https://github.com/lukasschwab/arxiv.py/issues/134

This is one from the orignal author investigating issues with it
https://github.com/lukasschwab/arxiv.py/issues/129

____
**TLDR :** 
it would word 30 percent of the time  , but then it would not load any results at all 
so  i spend a whole week investigating this 
i wanted to extend this project  such that  , 
- you read paper abstracts 
- you downlaod the paper n get text from it 
- AI model running inside of collab helps you summarize the paper 

-----
![image](https://github.com/bilalazh/Arxiv-Research-Pooler/assets/139261053/8619419c-7f44-4deb-8709-5dcc34e4bb23)


For now im  not sure what is happening with this whole arxiv api thing  , if this gets resolved ill extend this and add more features 
im using this for now 
https://bytez.com/



 

> downloading all arxiv in a specific category e.g.: biology to local folder

I am now locally using this https://github.com/MarkHershey/arxiv-dl on my ubtuntu machine to downloads papers 

ive set the alias to $paper
so it works like

`$paper arxiv-id1  arxiv-id 2 ......`

- as far as i know bulk donwload of too many papers is not recommended to overwhelm their API

**These links might  be useful to you** 
https://academia.stackexchange.com/questions/59093/bulk-download-of-arxiv-or-other-publication-data-set-with-metadata-and-citatio

https://info.arxiv.org/help/bulk_data/index.html


if you have any other questions /ideas to share feel free to do so 
thanks  , have a great day
